docs/markdown/snippets/rust_proc_macro_crates.md

```diff
@@ -0,0 +1,16 @@
+## Rust proc-macro crates
+
+Rust has these handy things called proc-macro crates, which are a bit like a
+compiler plugin. We can now support them, simply build a *shared_library* with
```

Comment:
We should be able to use `[[shared_library]]` here too.

Reply:
Can we? I thought that only worked from yank?

Reply:
No, it's not handled by the refman generator. It's handled by:
```meson
    extra_extension: meson.current_source_dir() / 'extensions' / 'refman_links.py',
```